src/core/client/ui/components/Message/Message.tsx

```diff
@@ -0,0 +1,75 @@
+import cn from "classnames";
+import React, { ReactNode, StatelessComponent } from "react";
+
+import { withStyles } from "talk-ui/hocs";
+
+import Icon from "../Icon";
+import * as styles from "./Message.css";
+
+export interface MessageProps {
+  /**
+   * The content of the component.
+   */
+  children: ReactNode;
+  /**
+   * Convenient prop to override the root styling.
+   */
+  className?: string;
+  /**
+   * Override or extend the styles applied to the component.
+   */
+  classes: typeof styles;
+  /*
+  * If set renders a full width message
+  */
+  fullWidth?: boolean;
+  /*
+  * Name of the icon, render if provided
+  */
+  icon?: string;
+  /*
+  * Name of color, "info" stays by default - common gray one
+  */
+  color?: "validation" | "info";
```

Comment:
Rather than having a new set of new enums that maps to our theme colors, it'd be simpler to just pass in the colors from our color palette of our theme `grey | primary | secondary | error`. The Button, Typography, TextField components are also refering to these colors.

Reply:
Agreed

Reply:
👍